tests/beaker/tests/create_pv_w_param_force.rb

```diff
@@ -0,0 +1,38 @@
+require 'master_manipulator'
+require 'lvm_helper'
+test_name "FM-4579 - C96646 - create physical volume without parameter 'name'"
```

Comment:
Name of the test file does not make sense considering the name of the test case.